.gitignore

```diff
@@ -1 +1,2 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
```

Comment:
I am not a fan of keeping an OS specific or IDE specific gitignore file in the remote repo. Perhaps you can add these in your local config, but don't commit to the remote?

Reply:
Addressed ✅